### ISSUE TITLE
FR: Crashlytics stable release ETA

### ISSUE THREAD

**Issue by anushenoy** [firebase/firebase-ios-sdk#5073]
Now that Fabric is being deprecated and the Firebase Crashlytics beta sdk has been out for quite some time,  when can we expect a stable release of the same. 

* Firebase Component: Crashlytics

**Comment by TKBurner** [comment]
Thanks we expect Fabric to be deprecated before the general release of our Firebase SDKs. They will end up being relatively close in timing, but it should be noted that we will continue support for the Fabric SDKs for many months after the Firebase SDKs have general release.

**Comment by kondratk** [comment]
@TKBurner 
Today I've received an email from Google "Fabric shutdown extended to Ma‌y 4, 2020".

Does it mean, that after shutting down Fabric service- Fabric SDKs will still continue to work and to comunicate with Firebase services like Crashlytics? Or is migration to Firebase SDK necessary?

I'm asking because right now we are not able to migrate because of that issue, which seems like a more serious problem: https://github.com/firebase/firebase-ios-sdk/issues/5020

**Comment by kkosovsky** [comment]
@TKBurner 
How does providing Carthage support for FirebaseCrashlytics fit in your release plans?
I don't want to use Cocoapods, I'm currently using some Firebase features in my project including vision and analytics, wanted to add Crashlytics and all I managed to do is adding locally my own custom json for Carthage, so it can download Fabric and Crashlytics. It's a temporary solution. 

```
we will continue support for the Fabric SDKs for many months after the Firebase SDKs have general release.
```
- How many is `many`? 6, 12, 18, 24? 

All the best!

**Comment by paulb777** [comment]
@kkosovsky See https://github.com/firebase/firebase-ios-sdk/blob/master/Carthage.md#carthage-usage for info on the current experimental support.

**Comment by kkosovsky** [comment]
@paulb777 
Yup, been there, done that :) Can you, please, answer my question?

Thank you

**Comment by erikkerber** [comment]
@kkosovsky You probably have dealt with this already, but with Carthage you have to look under the covers and figure out what transitive dependencies Firebase introduced with FirebaseCrashlytics.

From my experimentation:

- FirebaseInstallations
- PromisesObjC

**Comment by TKBurner** [comment]
Thanks, the anticipated SDK deprecation date for the Fabric SDKs is approximately Q4 2020.

**Comment by erikkerber** [comment]
@TKBurner Originally you stated:

> Thanks we expect Fabric to be deprecated before the general release of our Firebase SDKs.hey will end up being relatively close in timing

Since this was pushed out to May 4th from March 31st, do you imagine the first "stable" release of FirebaseCrashlytics would be soon?

Before the delay to May 4th, your previous statement made it sound as if the beta tag would come off relatively close to next week.

**Comment by TKBurner** [comment]
Yes, this is still on track. It is important to note it is tied to Firebase SDK releases, so if it doesn't ship in the next release it may be delayed by up to a month.

**Comment by anushenoy** [comment]
Hey @TKBurner,
Even in Firebase 6.22.0 i could see that the beta tag for crashlytics still hasnt come off.  Is there a revised ETA for the stable version?

**Comment by erikkerber** [comment]
6.23.0 has marked FirebaseCrashlytics available for general use.

https://github.com/firebase/firebase-ios-sdk/releases/tag/6.23.0

@anushenoy, I believe this can be closed.

**Comment by anush-shenoy** [comment]
Yup. Thanks a lot for the heads up

**Comment by paulb777** [comment]
Marked with the Released 6.23.0 milestone
